weblate/trans/tests/test_selenium.py

```diff
@@ -119,8 +119,8 @@ def setUpClass(cls):
 
         # Force English locales, the --lang and accept_language settings does not
         # work in some cases
-        backup_lang = os.environ["LANG"]
         os.environ["LANG"] = "en_US.UTF-8"
+        backup_lang = os.environ["LANG"]
```

Comment:
This does not make it reset back after completing test. Whnat was the issue here?

Reply:
I had to add `export LANG="en_US.UTF-8"` before running the test. I am not sure if it was needed.

Reply:
Please remove this from the pull request, I've addressed this in 7378e561b1.